modules/s2geometry/0.12.0-bcr.20250201-7c96080/presubmit.yml

```diff
@@ -0,0 +1,23 @@
+matrix:
+  platform:
+    - debian10
+    - ubuntu2004
+    - macos
+    - macos_arm64
+  bazel:
+    - 7.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+      - "--cxxopt=-std=c++17"
+      - "--host_cxxopt=-std=c++17"
+    build_targets:
+      - "@s2geometry//:s2"
+    test_flags:
+      - "--cxxopt=-std=c++17"
+      - "--host_cxxopt=-std=c++17"
+    test_targets:
+      - "@s2geometry//:*"
```

Comment:
Usually we don't specify the whole test suite for the module in the BCR presubmit.yml file. And looks like there are some test failures.